namespaces/live-1.cloud-platform.service.justice.gov.uk/offender-management-preprod/resources/rds-allocations-api.tf

```diff
@@ -0,0 +1,52 @@
+/*
+ * When using this module through the cloud-platform-environments, the following
+ * two variables are automatically supplied by the pipeline.
+ *
+ */
+
+variable "cluster_name" {}
+
+variable "cluster_state_bucket" {}
+
+/*
+ * Make sure that you use the latest version of the module by changing the
+ * `ref=` value in the `source` attribute to the latest version listed on the
+ * releases page of this repository.
+ *
+ */
+module "allocation-rds" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-rds-instance?ref=4.3"
+
+  cluster_name           = "${var.cluster_name}"
+  cluster_state_bucket   = "${var.cluster_state_bucket}"
+  team_name              = "offender-management"
+  db_instance_class      = "db.m4.large"
+  business-unit          = "HMPPS"
+  application            = "offender-management-allocation-manager"
+  is-production          = "true"
+  environment-name       = "preprod"
+  infrastructure-support = "omic@digital.justice.gov.uk"
+  db_engine              = "postgres"
+  db_engine_version      = "10"
+  db_name                = "allocations"
+  aws_region             = "eu-west-2"
```

Comment:
Replace `aws_region` with the `providers` block here please.

Example : https://github.com/ministryofjustice/cloud-platform-environments/blob/master/namespaces/live-1.cloud-platform.service.justice.gov.uk/prison-visits-booking-production/resources/rds-prison-visits.tf

Reply:
Done